contract/bridge/syscall_service.go

```diff
@@ -45,12 +45,27 @@ func (c *SyscallService) QueryBlock(ctx context.Context, in *pb.QueryBlockReques
 		return nil, err
 	}
 
-	blockbuf, err := json.Marshal(block)
-	if err != nil {
-		return nil, err
+	transactions := []*pb.Transaction{}
+	for _, t := range block.Transactions {
+		transaction := ConvertTxToSDKTx(t)
+		transactions = append(transactions, transaction)
```

Comment:
Since sdk only uses txid, there is no need to pass the whole tx here.

Reply:
Shall we use original txid or hextxid stored here?